addons/io/org.openhab.io.homekit/README.md

```diff
@@ -71,34 +65,34 @@ A full list of supported accessory types can be found in the table below.
  </tr>
  <tr>
   <td>&nbsp;</td>
-  <td>currentTemperature</td>
+  <td>CurrentTemperature</td>
   <td>Number</td>
-  <td>The current temperature, using the same unit rules as TemperatureSensor</td>
+  <td>The current temperature, same as above</td>
  </tr>
  <tr>
   <td>&nbsp;</td>
-  <td>heatingCoolingMode</td>
+  <td>homekit:HeatingCoolingMode</td>
```

Comment:
> You mean the unit? This will be addressed by eclipse/smarthome#601.

Hopefully that work will get revived soon.

> Right, but I am not yet at all clear on how I will model this exactly in the ontology (incl. naming). And since it is then anyhow likely to change again in future, I preferred to keep it here for the moment.

No problem. Just for reference, the Alexa api has 'temperatureMode' which can be ' AUTO, COOL, HEAT'  .  It seems this is a common pattern.